src/main/java/com/raoulvdberge/refinedstorage/gui/grid/sorting/GridSortingInventoryTweaks.java

```diff
@@ -0,0 +1,33 @@
+package com.raoulvdberge.refinedstorage.gui.grid.sorting;
+
+import com.raoulvdberge.refinedstorage.apiimpl.network.node.NetworkNodeGrid;
+import com.raoulvdberge.refinedstorage.gui.grid.stack.GridStackItem;
+import com.raoulvdberge.refinedstorage.gui.grid.stack.IGridStack;
+import net.minecraftforge.fml.common.Loader;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+
+/**
+ * A GridSorting implementation for the Inventory Tweaks sorting mod
+ * @author Cooliojazz
+ */
+public class GridSortingInventoryTweaks extends GridSorting {
+
+    @Override
+    public int compare(IGridStack o1, IGridStack o2) {
+        try {
+            invtweaks.api.InvTweaksAPI api = (invtweaks.api.InvTweaksAPI)Class.forName("invtweaks.forge.InvTweaksMod", true, Loader.instance().getModClassLoader()).getField("instance").get(null);
```

Comment:
At least save the return value of Class.forName in a field, since that call is performance intensive.